static/app/views/dashboardsV2/widgetBuilder/widgetBuilder.tsx

```diff
@@ -306,10 +307,6 @@ function WidgetBuilder({
         }
       }
 
-      if (prevState.dataSet === DataSet.ISSUES) {
```

Comment:
Removed this because it was causing the selection of the prebuilt Issues widget to flip back to the events dataset